IBPSA/Airflow/Multizone/BaseClasses/TwoWayFlowElement.mo

```diff
@@ -9,8 +9,16 @@ partial model TwoWayFlowElement "Flow resistance that uses the power law"
     final m2_flow_nominal=m1_flow_nominal);
   extends IBPSA.Airflow.Multizone.BaseClasses.ErrorControl;
 
-  replaceable package Medium = Modelica.Media.Interfaces.PartialMedium
-    annotation (choicesAllMatching=true);
+  replaceable package Medium =
+    Modelica.Media.Interfaces.PartialMedium "Medium in the component"
+      annotation (choices(
+        choice(redeclare package Medium = IBPSA.Media.Air "Moist air"),
+        choice(redeclare package Medium = IBPSA.Media.Water "Water"),
+        choice(redeclare package Medium =
```

Comment:
@JayHuLBL : Why do you allow the multizone airflow models to be configured with water or glycol? I can't see a use case where this makes sense.

Reply:
@mwetter I will further change the settings, as well as other inappropriate changes.